project-reports/2024/2024-annual-Hyperledger-Indy.md

```diff
@@ -0,0 +1,98 @@
+* The deprecation of the Indy SDK.
+
+In addition, as mentioned earlier in the report, significant progress was made in creating an Indy-Besu proof of concept.
+
+### Next Year’s Goals
+
+_Guidance: What are the goals for the next year of the project? The goals should list what you want to achieve, not just what you know you can achieve. Feel free to include stretch goals and things that you are looking to explore in the next year. For example, are you working on major new features? Or are you concentrating on adoption, community growth, or documentation?_
+
+The goals for 2024 derive from both 2023 Indy Roadmap document, and the outcome of the Indy Ecosystems Summits held in 2023. The bullet point versions of these goals include the following. The Indy maintainers and contributors have a deeper understanding of what is behind these bullet points:
+
+* Increased awareness of Indy, its usefulness, and a corresponding increase in contributors to enable it to continue to evolve.
+* Formally release the Ubuntu 20.04 release of Indy Node.
+    * Look into updating the solution to later releases.
+* Tools and guidance in updating existing networks to the 20.04 release.
+* Continue to support the migration of Indy clients away from the Indy SDK.
+    * Update the Indy documentation to point people to current “best practices”.
+    * Update and release the Indy VDR component as needed.
+* Address the “Corporate Firewall” issue, where users behind a corporate firewall cannot access an Indy network because of the ZMQ ports being used.
+* Evolve Indy so as to be ready for when the current implementation will need more functionality, such via the Indy-Besu approach.
+* Desired functionality:
+    * Tombstoning
+    * Additional DIDDoc support, such support for additional key types
+    * Read replicas for performance
+    * Storing other document types on an Indy ledger, such as OCA Bundles
+
+### Help Required
+
+_Guidance: How can the Hyperledger Foundation or the TOC help you achieve your upcoming goals?_
+
+No specific requests at this time. There is a lot that the Hyperledger Foundation provides that we need to be making use of, particularly in the area of marketing / getting the word out about the capabilities of Hyperledger Indy. Our challenge is that the project is driven largely by technical types, not marketers.
+
+## Project Lifecycle Stage Recommendation
+
+_Guidance: What stage do you think the project should be? If you think that your project meets the criteria for another stage, please explain why. (See [Project Lifecycle](https://github.com/hyperledger/toc/blob/gh-pages/governing-documents/project-lifecycle.md))_
+
+Indy is a Graduated project and should remain so. Its code is rock solid, and a relatively large ecosystem of network operators, maintainers, contributors, and users.
```

Comment:
Overall, I think this is great and tracks with the project maintaining its status. Apologies in advance for harping on Indy-Besu :)